README.md

```diff
@@ -218,6 +218,22 @@ Currently, the following cases are available for this task:
 - Go
   - `plain/plain.go`: An empty function that does nothing.
```

Comment:
We have more than the plain examples by now. Please add a commit where they are added `with pretty links`.